baseband/mark5b/header.py

```diff
@@ -147,12 +183,12 @@ def infer_kday(self, ref_time):
     @property
     def payloadsize(self):
         """Size of the payload, in bytes."""
-        return 10000  # 2500 words
+        return self._payloadsize    # Hardcoded in class definition.
 
     @payloadsize.setter
     def payloadsize(self, payloadsize):
         if payloadsize != 10000:  # 2500 words
```

Comment:
I'm not completely sure of the need for a private attribute, but going that route we should also change it here.